core/linux.go

```diff
@@ -86,18 +89,19 @@ var copyRule = pctx.StaticRule("copy",
 type targetableModule interface {
 	blueprint.Module
 	getTarget() string
+	outputName() string
```

Comment:
I think this targetableModule interface should be redefined to be based on a new interface (instead of blueprint.Module) where it would get outputName.

Reply:
Moved to new interface but name of it is still under discussion